src/client/mixins/basic-interaction.ts

```diff
@@ -67,6 +68,16 @@ export class BasicInteractionAware {
     }
   }
 
+  public async pressKey(key: string) {
+    if (!keyCodes[key]) throw Error('Key is invalid');
+
+    try {
+      await this.client.keyboard.type(String.fromCharCode(keyCodes[key]));
+    } catch (e) {
+      throw Error('There was an error pressing the key');
```

Comment:
I think the error `e` here should be rethrown to the step i.e. don't create another Error object